src/time.rs

```diff
@@ -47,9 +44,7 @@ impl Time {
     /// ```
     #[cfg(feature = "std")]
     pub fn try_from(time: std::time::SystemTime) -> Result<Time, ring::error::Unspecified> {
```

Comment:
Go ahead and just remove this function.

Reply:
Sounds good; I kept just in case needing to add `use std::convert::TryFrom` would be considered a breaking change